docs: correct third-party plugin support on plugins overview page (#991)
dmno-dev/varlock: packages/varlock-website/src/content/docs/plugins/overview.mdx

Plugins allow extending the functionality of Varlock. See the [plugins guide](/guides/plugins/) for more details on using plugins.

> The plugins listed below are the official ones, published under the `@varlock` npm scope. Third-party plugins are also supported. You can load them from npm or from a local path, and the [plugins guide](/guides/plugins/#installation) covers authoring and installing your own. Loading plugins from other sources (git, http, jsr) is not supported yet.

Author: ```suggestion
The plugins listed below are the official ones, published under the `@varlock` npm scope. Third-party plugins are also supported. You can load them from npm or from a local path. See the [plugins guide](/guides/plugins/#installation) for more information. Loading plugins from other sources (git, http, jsr) is not supported yet.
```
